src/config_handling.py

```diff
@@ -30,7 +35,10 @@ def init_config(app):
     )
 
     for config in all_configs:
-        value = os.environ.get(config, get_default_for(config))
+        if config in file_configs:
+            value = get_config_from(app, config + "_FILE")
+        else:
+            value = os.environ.get(config, get_default_for(config))
         if not value:
             continue
```

Comment:
Raise here an exception respectively exit the program and delete the lines 54 - 57.

Reply:
Throw an error and exit the program here.